README.md

```diff
@@ -5,44 +5,51 @@
-| [Kubeflow Notebooks](https://www.kubeflow.org/docs/components/notebooks/)                   | [`kubeflow/notebooks`](https://github.com/kubeflow/notebooks)                 |
-| [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/)                   | [`kubeflow/pipelines`](https://github.com/kubeflow/pipelines)                 |
-| [Kubeflow Spark Operator](https://www.kubeflow.org/docs/components/spark-operator/)         | [`kubeflow/spark-operator`](https://github.com/kubeflow/spark-operator)       |
-| [Kubeflow Training Operator](https://www.kubeflow.org/docs/components/training/)            | [`kubeflow/training-operator`](https://github.com/kubeflow/training-operator) |
+| Kubeflow Tool                                                                               | Source Code                                                             |
+| ------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------- |
+| [KServe](https://www.kubeflow.org/docs/external-add-ons/kserve/)                            | [`kserve/kserve`](https://github.com/kserve/kserve)                     |
+| [Kubeflow Katib](https://www.kubeflow.org/docs/components/katib/)                           | [`kubeflow/katib`](https://github.com/kubeflow/katib)                   |
+| [Kubeflow Model Registry](https://www.kubeflow.org/docs/components/model-registry/)         | [`kubeflow/model-registry`](https://github.com/kubeflow/model-registry) |
+| [Kubeflow MPI Operator](https://www.kubeflow.org/docs/components/training/user-guides/mpi/) | [`kubeflow/mpi-operator`](https://github.com/kubeflow/mpi-operator)     |
+| [Kubeflow Notebooks](https://www.kubeflow.org/docs/components/notebooks/)                   | [`kubeflow/notebooks`](https://github.com/kubeflow/notebooks)           |
+| [Kubeflow Pipelines](https://www.kubeflow.org/docs/components/pipelines/)                   | [`kubeflow/pipelines`](https://github.com/kubeflow/pipelines)           |
+| [Kubeflow Spark Operator](https://www.kubeflow.org/docs/components/spark-operator/)         | [`kubeflow/spark-operator`](https://github.com/kubeflow/spark-operator) |
+| [Kubeflow Trainer](https://www.kubeflow.org/docs/components/trainer/)                       | [`kubeflow/trainer`](https://github.com/kubeflow/trainer)               |
 
-## Kubeflow Platform
+## Kubeflow Reference AI Platform
 
-The [Kubeflow Platform](https://www.kubeflow.org/docs/started/introduction/#what-is-kubeflow-platform) refers to the full suite of Kubeflow Components bundled together with additional integration and management tools.
+The [Kubeflow Reference AI Platform](https://www.kubeflow.org/docs/started/introduction/#what-is-kubeflow-platform)
+refers to the full suite of Kubeflow Tools bundled together with additional integration and management tools.
```

Comment:
i think multi-user/tenancy, authentication and authorization is missing here.

Reply:
It might be tricky to explain since multi-tenancy means different meaning for various tools (e.g. KFP, Workspaces, Trainer).
I guess additional integration and management tools cover that point, don't you think so @juliusvonkohout ?

Additionally, we could have website page which explains in details:
What is Kubeflow Reference AI Platform.